plugins/module_utils/errors.py

```diff
@@ -180,44 +187,91 @@ def format_failed_fusion_operation(op):
     return output
 
 
+def format_http_exception(exception, traceback):
+    """Formats failed `urllib3.exceptions` exceptions into a simple short form,
+    suitable for Ansible error output. Returns a `str`."""
```

Comment:
`def format_http_exception(exception, traceback) -> str:`

Reply:
Ansible broke for me on type hints, unfortunately